fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBRecordContextConfig.java

```diff
@@ -344,7 +354,7 @@ public Builder setTransactionTimeoutMillis(long transactionTimeoutMillis) {
             return this;
         }
 
-        private long getTransactionTimeoutMillis() {
+        public long getTransactionTimeoutMillis() {
```

Comment:
I suppose this should have a Javadoc comment along the lines of the other getters in this builder

Reply:
Added